src/ZiggyCreatures.FusionCache.Backplane.NATS/docs/README.md

```diff
@@ -0,0 +1,13 @@
+﻿# FusionCache
+
+![FusionCache logo](https://raw.githubusercontent.com/ZiggyCreatures/FusionCache/main/docs/logo-256x256.png)
+
+### FusionCache is an easy to use, fast and robust hybrid cache with advanced resiliency features.
+
+It was born after years of dealing with all sorts of different types of caches: memory caching, distributed caching, http caching, CDNs, browser cache, offline cache, you name it. So I've tried to put together these experiences and came up with FusionCache.
+
+Find out [more](https://github.com/ZiggyCreatures/FusionCache).
+
+## 📦 This package
+
+This package is a backplane implementation on [NATS](https://nats.io/) based on the awesome [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis) library.
```

Comment:
May be I miss something but how NATS implementation could be based on Redis library. I can understand that this library was used as prototype or source of inspiration but not based on this.